app/ai/voice/agents/breeze_buddy/managers/utils.py

```
@@ -1,14 +1,34 @@
 from app.services.redis.client import get_redis_service
 
 
+def is_within_calling_hours(config: CallExecutionConfig) -> bool:
+    """
+    Checks if the current time is within the allowed calling hours.
+    """
+    IST = timezone(timedelta(hours=5, minutes=30))
+    current_time = datetime.now(IST).time()
+
+    if config.call_start_time <= config.call_end_time:
+        # Normal case (e.g., 09:00–17:00)
```

Ruff (0.14.13)

Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)